test/unit/HoldConfirmation.test.js

```diff
@@ -4,7 +4,7 @@ import sinon from 'sinon';
 import { expect } from 'chai';
 import { mount } from 'enzyme';
 // Import the component that is going to be tested
-import HoldConfirmation from './../../src/app/components/HoldPage/HoldConfirmation.jsx';
```

Comment:
This PR is not going to write/update the test to pass `test/unit/HoldConfirmation.test.js`, is it correct?